lib/buildsystems/autotools/v1.rb

```diff
@@ -0,0 +1,43 @@
+require 'package'
+
+class Autotools_v1 < Package
+  @_buildsystems_autotools_version = '1.0.0'
+
+  def self.configure_options (options = '')
+    if options
+      @configure_options = options
+    end
+    return @configure_options
+  end
+
+  def self.check? (bool = false)
+    @check = bool
+    return @check
+  end
+
+  def self.build
+    # Run autoreconf if necessary
+    unless File.executable? './configure'
```

Comment:
Would it not make more sense to regenerate no matter what? That would, among other things, remove the need for `filefix` (after we update to libtool 2.4.7, that is).

Reply:
I want to say yes, however we've had some instances where packages compile from tarballs with a ./configure script already there, however have failed when re`autoreconf`ing or `autoreconf`ing from a git repository. Maybe a variable like `autoreconf?` that normally defaults to false but can be set to true if need be, something like `svem?` as seen in `lib/buildsystems/setup_py/v1.rb`

Reply:
If it fails, then that's probably an upstream issue which we can report or update to fix. A variable sounds good as a fallback, but I'd prefer to keep the average package as small as possible.

Reply:
Maybe something that backs up `./configure` and friends, then runs autoreconf, restoring the original backups if it fails?

Reply:
that honestly sounds like too much effort for the job we're trying to do here: specific versions of packages are meant to be built (and have been tested with) certain versions of autotools. If we use a different version, we're introducing more instability unnecessarily into a system that, not to poop on any of the devs, myself, included, but chromebrew is quite unstable.

Reply:
see e5ce5a1